time_sequence_prediction/train.py

```diff
@@ -15,12 +15,12 @@ def __init__(self):
         self.lstm2 = nn.LSTMCell(51, 51)
         self.linear = nn.Linear(51, 1)
 
-    def forward(self, input, future = 0):
+    def forward(self, input, device, future = 0):
         outputs = []
-        h_t = torch.zeros(input.size(0), 51, dtype=torch.double)
-        c_t = torch.zeros(input.size(0), 51, dtype=torch.double)
-        h_t2 = torch.zeros(input.size(0), 51, dtype=torch.double)
-        c_t2 = torch.zeros(input.size(0), 51, dtype=torch.double)
+        h_t = torch.zeros(input.size(0), 51, dtype=torch.double).to(device)
```

Comment:
typically it's better to `model.to(device)` the entire module if you see yourself doing this